### PR TITLE
Online text editor with  collaboration 

### PR DESCRIPTION
**Pull request by jsroyal**
http://www.writeurl.com/

**A collaborative real-time online text editor.No registration. No password. Just click new document.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- [x] I have read the [Contribution guidelines](https://github.com/aviaryan/awesome-no-login-web-apps/blob/master/CONTRIBUTING.md) and I am confident that my PR is suitable. 
- [x] This pull request has a descriptive title. For example, `Add [App name]`, not `Update readme.md` or `Added new entries`.
- [x] The app I added is not a duplicate.

### CODE REVIEW

**Comment by Anshuman-Verma** [comment]
The PR heading can be improved, also the markdown in PR description is not used correctly @jsroyal sir.

**Comment by jsroyal** [comment]
What is cause for?

>The Travis CI build failed

**Comment by aviaryan** [comment]
Not with your PR. Check travis log.

File: README.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ To save the world from creating user accounts and installing software applicatio
 * [EtherCalc](https://ethercalc.net/) - Online spreadsheet editor, exports to Excel.
 * [PdfEscape](https://www.pdfescape.com/) - Edit or create PDFs in browser itself.
 * [Browserpad](http://browserpad.org/) - A server-less plain text editor in the browser. Allows you to open and save plain text files.
+* [WriteURL](http://www.writeurl.com/) - A collaborative real-time online text editor.No registration. No password. Just click new document.
```
Comment (aashutoshrathi) [inline README.md]: Description can be improved to -
> A collaborative real-time online text editor. No registration, No password and Simple sharing. Just click [new document.](http://www.writeurl.com/new)

**Comment by jsroyal** [comment]
@aviaryan I not getting what is an issue with it. Can you see the log:100:

**Pull request merged**